src/cow-react/modules/swap/services/ethFlow/steps/calculateUniqueOrderId.ts

```diff
@@ -7,6 +7,7 @@ import { getOrderParams, PostOrderParams } from 'utils/trade'
 import { getDomain } from 'utils/signatures'
 import { MAX_VALID_TO_EPOCH } from '@cow/utils/time'
 import { WRAPPED_NATIVE_CURRENCY } from 'constants/tokens'
+import { Order } from '@cowprotocol/contracts/src/ts/order'
```

Comment:
Shouldn't this come from the SDK?

Just mentioning because in the next PR you are loading the contracts dynamically but here it's being imported directly which I think invalidates the follow up changes

Reply:
Thank you!
Fixed imports:
https://github.com/cowprotocol/cowswap/pull/2104/commits/c72bf1f77e95b908ed7b5a50f8572eede0c4972a